{{ cookiecutter.package_name|slugify }}/docker-compose.yml

```diff
@@ -12,7 +12,12 @@ services:
         {%- endif %}
       context: .
       target: dev
-    command: [ "sleep", "infinity" ]
+    command:
+      [
+        "sh",
+        "-c",
+        "cp --update /var/lib/poetry/poetry.lock && cp --update /var/lib/git/* /app/.git/hooks/ && sleep infinity"
```

Comment:
I think you're missing the destination for the first `cp` here.

Reply:
Oh, good catch, got lost in my edit.

Reply:
Fixed!